src/entry_point_win.cpp

```diff
@@ -113,7 +113,13 @@ int WINAPI wWinMain(HINSTANCE /*hInstance*/, HINSTANCE /*hPrevInstance*/, LPWSTR
 		signal(SIGABRT, signal_abort_handler);
 	}
 
-	SetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2);
+	// Workaround for old machines
+	HINSTANCE hUser32dll = LoadLibrary(L"User32.dll");
+	if(hUser32dll) {
+		auto pSetProcessDpiAwarenessContext = (decltype(&SetProcessDpiAwarenessContext))GetProcAddress(hUser32dll, "SetProcessDpiAwarenessContext");
+		pSetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2);
```

Comment:
this isn't going to fix anything. On old windows, the dll won't contain the function (and even if it did, wouldn't understand the constant value). Nor can you simply omit it without things behaving badly. You would need to fall back to an older version of dpi awareness and rewrite the per-monitor sizing routine for that instead.